packages/mc-dev-authentication/package.json

```diff
@@ -28,10 +28,10 @@
     "@babel/runtime-corejs3": "^7.20.13"
   },
   "devDependencies": {
-    "@commercetools-frontend/application-config": "workspace:*",
-    "@types/express": "^4.17.17",
+    "@commercetools-frontend/application-config": "22.6.0",
```

Comment:
Just double checking if changing this dependency from workspace to an specific one intended

Reply:
Ah this is a dev dep...yeah sorry we can keep using `workspace`, I'll revert